### PR TITLE
Add Formik form for creating and updating Properties

### PR DESCRIPTION
**Pull request by cdpadilla42**
Add components for a page displaying a form that can be used flexibly for updating and posting new properties. Next steps include incorporating Apollo mutations and handling existing properties updating.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+TenantFile/Api/tenant-file-12264437b25e.json
```
Comment (ryanrightmer) [inline .gitignore:1]: What was this for? Was this file the side effect of something?

Also, the ionide file should probably be added.

**Review by ryanrightmer — COMMENTED** [review]
Looks good as an intermediate step. I'm assuming a follow up is planned to finish adding the mutation and backend logic. Just the one gitignore file comment and I'm good to merge.

File: .gitignore (hunk 1 above)
Comment (cdpadilla42) [reply .gitignore:1]: That file was created for holding Google Cloud authorization as per one of the methods, but it ended up not being needed. Deleting it from my local repository and from the .gitignore for the next commit.

**Review by ryanrightmer — APPROVED** [review]
Works for me

**Pull request merged**